udp_mux.go

```diff
@@ -204,6 +219,40 @@ func (m *UDPMuxDefault) createMuxedConn(key string) *udpMuxedConn {
 	return c
 }
 
+func (m *UDPMuxDefault) handleRemotePing(msg *stun.Message) (*udpMuxedConn, error) {
+	attr, err := msg.Get(stun.AttrUsername)
+	if err != nil {
+		return nil, err
+	}
+
+	ufrag := strings.Split(string(attr), ":")[0]
```

Comment:
Will this segfault if an attacker sends a message without `:` ?

Reply:
No, because `strings.Split` always returns a slice with 1 element, even for an empty string.

Reply:
:+1: